plugins/modules/nfs_global_settings.py

```diff
@@ -197,12 +205,33 @@ def __init__(self):
 
         self.api_client = utils.get_powerscale_connection(self.module.params)
         self.isi_sdk = utils.get_powerscale_sdk()
+        cluster_api = utils.isi_sdk.ClusterApi(self.api_client)
+        major = str(cluster_api.get_cluster_config().to_dict()['onefs_version']['release'].split('.')[0])
+        minor = str(cluster_api.get_cluster_config().to_dict()['onefs_version']['release'].split('.')[1])
+        self.array_version = major + "." + minor
```

Comment:
No need to get the version again. You may get it directly from. I added in the last commit.
```
major = self.isi_sdk.major
minor = self.isi_sdk.minor
```